radixdlt-java/src/test/java/com/radixdlt/client/core/atoms/AtomTest.java

```diff
@@ -5,8 +5,11 @@
 import static org.junit.Assert.assertTrue;
 
 import com.radixdlt.client.core.atoms.particles.Spin;
+import com.radixdlt.client.core.util.Hash;
+import okio.ByteString;
```

Comment:
Again, seems like an unnecessary additional dependency on an external library.  Can we use Bytes.toHexString(...) instead?